dotcom-rendering/.prettierignore

```diff
@@ -23,6 +23,8 @@ node_modules/
 !/fixtures/**/*.ts
 !/cypress/**/*.js
 !/cypress/**/*.ts
+!/cdk/**/*.js
+!/cdk/**/*.ts
```

Comment:
Why don't we lint this lot?

Reply:
Remnants of when I had CDK as its own package. Fixed!